src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -74,6 +74,35 @@ public virtual async Task<(bool, FunctionDescriptor)> TryCreate(FunctionMetadata
             }
         }
 
+        private bool BindingAttributeContainsExpression(IEnumerable<FunctionBinding> bindings)
+        {
+            if (bindings.Count() <= 0)
+            {
+                return false;
+            }
+
+            var bindingExpressionRegex = new Regex(@"{(.*?)\}");
+
+            foreach (ExtensionBinding binding in bindings)
+            {
+                if (binding.Metadata.IsTrigger)
+                {
+                    // skip triggers, we only care if input bindings contain expressions
+                    continue;
+                }
+
+                foreach (var attribute in binding.Attributes)
+                {
+                    if (bindingExpressionRegex.IsMatch(attribute.ToString()))
```

Comment:
this doesn't work, need to figure out how to get `BlobPath` from this attribute. We do know the type via TypeId

attribute.ToString() -> "Microsoft.Azure.WebJobs.BlobAttribute"

Reply:
Actually I just found GetAttributeData - I think this will solve the issue